code/datums/ruins/space.dm

```diff
@@ -182,3 +182,70 @@
 	name = "Crashed Wizard Shuttle"
 	description = "A shuttle of the Wizard Federation, sent out to crush some wandless scum. Unfortunately, the pilot suffered a magic-related accident and the shuttle crashed into a nearby asteroid."
 	cost = 2
+
+/datum/map_template/ruin/space/abandonedtele
+	id = "abandonedtele"
+	suffix = "abandonedtele.dmm"
+	name = "Abandoned Teleporter"
+	description = "An old teleporter, seemingly part of what used to be a larger satellite."
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/blowntcommsat
+	id = "blowntcommsat"
+	suffix = "blowntcommsat.dmm"
+	name = "Blown-out Telecommunications Satellite"
+	description = "The remains of an old telecommunications satellite once utilised by NanoTrasen. It lays derelict, with quite a few pieces missing."
+	cost = 5 // This is a chonky boy
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly. Its also huge.
+
+/datum/map_template/ruin/space/clownmime
+	id = "clownmime"
+	suffix = "clownmime.dmm"
+	name = "Clown & Mime Mineral Deposits"
+	description = "A crash site of two opposing factions, both trying to complete mining trips for their own valuable minerals. While all the crew have long perished, the minerals are likely intact."
+
+/datum/map_template/ruin/space/dj
+	id = "dj"
+	suffix = "dj.dmm"
+	name = "Russian DJ Station"
+	description = "An old russian listening station, long since defunct and lifeless, however the equipment is likely still in working condition."
+	cost = 2
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/druglab
+	id = "drublab"
+	suffix = "druglab.dmm"
+	name = "Drug Lab"
+	description = "An old abandoned \"Chemistry\" site, which has a strong aura of amphetamines around it."
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/syndiedepot
```

Comment:
since this is no longer guaranteed, does it need adjusting on its barrier? It would suck to have it spawned in and then inaccessible anyway, wouldn't it?

Reply:
It is guaranteed, read the lines
`228] always_place = TRUE // This is on the always spawn list because its already got a 50% chance to be shielded out, and may as well exist`

Reply:
You edited that in after I started viewing the file. I am onto you! *shakes fist